src/modules/75-cf/pages/feature-flags-detail/targeting-rules-tab/hooks/useFeatureEnabled.tsx

```diff
@@ -5,22 +5,27 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
+import { useMemo } from 'react'
 import useActiveEnvironment from '@cf/hooks/useActiveEnvironment'
 import { useFeature } from '@common/hooks/useFeatures'
 import { usePermission } from '@rbac/hooks/usePermission'
 import { FeatureIdentifier } from 'framework/featureStore/FeatureIdentifier'
 import { ResourceType } from '@rbac/interfaces/ResourceType'
 import { PermissionIdentifier } from '@rbac/interfaces/PermissionIdentifier'
 
-interface UseFeatureEnabledReturn {
+export interface UseFeatureEnabledReturn {
   enabledByPermission: boolean
   enabledByPlanEnforcement: boolean
   featureEnabled: boolean
   canEdit: boolean
   canToggle: boolean
 }
 
-const useFeatureEnabled = (): UseFeatureEnabledReturn => {
+export interface UseFeatureEnabledProps {
+  tags?: string[]
+}
+
+const useFeatureEnabled = ({ tags = [] }: UseFeatureEnabledProps): UseFeatureEnabledReturn => {
```

Comment:
Since this is the only thing in the args, might as well just pass it as itself instead of wrapping in an object. Means you can yoink the passing of empty objects.
```suggestion
const useFeatureEnabled = (tags: string[] = []): UseFeatureEnabledReturn => {
```